services/authentication/app.py

```diff
@@ -0,0 +1,93 @@
+import os
+
+from flask import Flask, request, jsonify, redirect
+
+from db import Base, engine
+from resources.login import Login
+from resources.roles_updates import RolesUpdates
+from resources.roles import Roles
+
+app = Flask(__name__)
+app.config["DEBUG"] = True
+Base.metadata.create_all(engine)
+
+@app.route('/login/<uuid>/<digid_message>', methods=['GET'])
```

Comment:
uuid is geen int. Dit is een uuid: d58f8562-022a-4569-b120-46e7c0a727a3 is dus een string. Die gebruik je om id's niet naar de buitenwereld te showen. 

En elke gebruiker heeft zijn eigen unieke login call? Wat gebeurd er als ik wil inloggen met een gebruiker die niet bestaat?

Reply:
will fix this